### PR TITLE
Csharp sdk

### PR DESCRIPTION
**Pull request by IridiumOxide**
### Description of change

@coderabbitai ignore

##### Checklist

- [ ] Tested in playground or other setup
- [ ] Screenshot (Grafana) from playground added to PR for 15+ minute run
- [ ] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

## Summary by CodeRabbit

- **New Features**
  - Introduced a new search functionality with a `Search` class for fetching and displaying results.
  - Added a search bar to the `Hero` component for improved user navigation.
  - Integrated a `Search` component into the `App` component to enhance user interaction.

- **Style**
  - Implemented new styles for the search bar to align with the application's design language.

- **Documentation**
  - Updated documentation to reflect the addition of the search feature and its user interface components.

### CODE REVIEW

File: sdks/aperture-csharp/Core/Utils.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using NodaTime;
+
+namespace ApertureSDK.Core;
+
+public static class Utils
+{
+    public static long GetCurrentEpochNanos()
+    {
+        var now = SystemClock.Instance.GetCurrentInstant();
+        return now.ToUnixTimeTicks() * 100;
```
[the remark this answers is not in the document]
Comment (IridiumOxide) [reply sdks/aperture-csharp/Core/Utils.cs:10]: But if a single tick represents 100 nanoseconds, we need to multiply the number of ticks by 100 to obtain the exact number of nanoseconds

[the remark this answers is not in the document]
Comment (IridiumOxide) [reply sdks/aperture-csharp/Core/Utils.cs:10]: You're really bad at math man

[the remark this answers is not in the document]
Comment (IridiumOxide) [reply sdks/aperture-csharp/Core/Utils.cs:10]: Ok imagine that: Alice has some boxes of apples, with 100 apples in each box. Let's say you can use an Alice.GetBoxesCount() method to retrieve the number of boxes of apples. What would be the expression to get the total number of apples?

[the remark this answers is not in the document]
Comment (IridiumOxide) [reply sdks/aperture-csharp/Core/Utils.cs:10]: yea, so time ticks are like boxes with 100 nanoseconds each, that's why we use `now.ToUnixTimeTicks()*100` to get total nanoseconds, same as apples

**Review by tanveergill — APPROVED** [review]

**Pull request merged**